app/models/katello/content.rb

```diff
@@ -0,0 +1,11 @@
+module Katello
+  class Content < Katello::Model
+    include Glue::Candlepin::ProductContent if SETTINGS[:katello][:use_cp]
+
+    has_one :product_content, :class_name => 'Katello::ProductContent', :dependent => :destroy
+    has_one :product, :through => :product_content
```

Comment:
after playing around with this, it does seem that a content could be associated to many products.  You can see this by importing two manifests with the same products into different orgs.  So i think this should be has_many and called 'product_contents'